Added warning about faulty aliexpress parts to smol tracker page (#518)
SlimeVR/SlimeVR-Docs-Site: src/smol-slimes/hardware/smol-tracker.md

### 📻 Microcontroller Boards

#### ProMicro nRF52840 {#ProMicro}

A clone of the **nice!nano** board. Cheapest option overall. Signal strength can be improved with an antenna mod.

> 

Reviewer: Altered formatting and added brand.
```suggestion
A common, cheap clone of the **Nice!Nano** board produced by Nice!. Signal strength may be improved with an antenna mod.
```

Things to take into account:

> - **DOA rates** (Dead On Arrival) - expect 10-20% or higher failure rates

Author: Bad phone case of 20% overage commonly recommended for those. Corrected.

- **Random failures** - Units fail unexpectedly after initial use, not just out of the box
- **Price** - due to how cheap they are, they are common part for DIY smol slimes.

Obtaining:
- Available on AliExpress with `compatible with nice!nano`, `SuperMini`, or `Pro Micro` branding.

> - [AliExpress TENSTAR 2pcs pack](https://pl.aliexpress.com/item/1005007738886550.html)

Reviewer: Added colour variants and significantly altered wording to improve readability. Also included links for both red and black board variants.
```suggestion
Parts are available on AliExpress, typically sold under "compatible with Nice!Nano", "Supermini", or "Promicro" branding. Availability of the red and black board variants appears to be dependent on region.

[2pcs AliExpress TENSTAR Red Nice!Nano board](https://www.aliexpress.com/item/1005007738886550.html)
[2pcs AliExpress Supermini Black board](https://www.aliexpress.com/item/1005007078900079.html)
```

Author: <img width="1230" height="482" alt="Image" src="https://github.com/user-attachments/assets/7eb86af7-0621-4051-a92f-5db308cb871d" />
Looks a little odd. Will alter

#### Seeed Studio XIAO nRF52840 {#XIAO}
A compact alternative board option.

Obtaining:
- [Manufacturer listing](https://www.seeedstudio.com/Seeed-XIAO-BLE-nRF52840-p-5201.html)

#### White Integrated AliExpress Boards (ICM-45686 + QMC6309)
Can be found under a variety of different names.

<img src="img/alirespess-white-aliexpress-smoll.webp" class="small-size-image" />

> Acceptable as a last resort, but expect quality and reliability issues. They can ship with unreliable firmware and may fail randomly, not just on arrival.

Reviewer: Community experience. I have a few of these, they are pretty terrible. The failure usually appears to be because the ICM-45686 and NRF52840 chips are salvaged parts that are several years old.

    </table>
</div>

###### White Aliexpress LSM6DSV + QMC6309 Modules

Can be found under variety of different names.

> ```admonish warning
> The magnetometer receives insufficient power and may brown out. Use as IMU only.
> ```

Reviewer: Clarified with up to date information from recent community work on enabling magnetometer support.
```suggestion
```admonish note
The QMC6309 magnetometer requires a minimum of 3v3 to function properly. Ensure that the correct voltage is provided to the 3v3 pin on the breakout board. Note that on Supermini/Promicro boards, the default SlimeVR bootloader provides 2v1, which is sufficient for the LSM6DSV IMU but not for the QMC6309 magnetometer.
```
```

Community-designed cases can be found on the  
[Smol Community Builds](./smol-slimes-community-builds.md) page.

### 🚫 Prebuilt AliExpress Trackers
<img src="img/alirespess-chrysalis-clone-tracker.webp" class="small-size-image">

> 

Reviewer: You should clarify why these are worse than regular chrysalis.
